tools/autodeployment/src/docker_events/index.js

```diff
@@ -0,0 +1,60 @@
+const shell = require('shelljs');
+const fetch = require('node-fetch');
+const Events = require('events');
+
+const { SLACK_SEND_MESSAGE } = process.env;
+
+const dockerEvents = new Events.EventEmitter();
+
+const message = {
+  response_type: 'in_channel',
+  text: `\n:rotating_light: PRODUCTION IS DOWN. :rotating_light:\n`,
+  attachments: [
+    {
+      text: `PROD IS DOWN`,
+    },
+  ],
+};
+
+const messageUp = {
+  response_type: 'in_channel',
+  text: `\nPRODUCTION IS BACK\n`,
+  attachments: [
+    {
+      text: `PROD IS UP!\n`,
+    },
+  ],
+};
+
+const waitingDocker = () => {
+  shell.chmod('700', './waitDocker.sh');
```

Comment:
This seems odd/dangerous to do in a script.  Can we not do it as part of installing the autodeployment server once?